models/articlesModel.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection");
+
+const fetchArticle = (article_id) => {
+  if (isNaN(article_id)) {
+    return Promise.reject({ status: 400, msg: "Invalid article_id query" });
```

Comment:
I think this would likely be handled by psql if you allowed it to be passed in, its kind of fine, but I think that its adding unnecessary logic on each successful request